pmlc/conversion/comp_to_llvm/comp_to_vulkan.cc

```diff
@@ -0,0 +1,476 @@
+// Copyright 2020, Intel Corporation
+#include "pmlc/conversion/comp_to_llvm/passes.h"
+
+#include "mlir/Dialect/GPU/GPUDialect.h"
+#include "mlir/Dialect/LLVMIR/LLVMDialect.h"
+#include "mlir/Dialect/SPIRV/SPIRVOps.h"
+#include "mlir/Dialect/SPIRV/Serialization.h"
+#include "mlir/Dialect/StandardOps/IR/Ops.h"
+#include "mlir/IR/Attributes.h"
+#include "mlir/IR/Builders.h"
+#include "mlir/IR/Function.h"
+#include "mlir/IR/Matchers.h"
+#include "mlir/IR/Module.h"
+#include "mlir/IR/PatternMatch.h"
+#include "mlir/IR/StandardTypes.h"
+#include "mlir/Support/LogicalResult.h"
+#include "mlir/Transforms/DialectConversion.h"
+
+#include "pmlc/conversion/comp_to_llvm/pass_detail.h"
+#include "pmlc/conversion/comp_to_llvm/utils.h"
+#include "pmlc/dialect/comp/ir/dialect.h"
+
+namespace pmlc::conversion::comp_to_llvm {
```

Comment:
I wonder whether we ought to have a separate namespace here?

Reply:
I think this is correct, it should follow the directory structure, and `comp_to_vulkan` lives in `comp_to_llvm`